packages/commonwealth/server/scripts/migrateChainEntities.ts

```diff
@@ -23,7 +22,7 @@ import { constructSubstrateUrl } from '../../shared/substrate';
 
 const log = factory.getLogger(formatFilename(__filename));
 
-const ENTITY_MIGRATION = process.env.ENTITY_MIGRATION;
+const CHAIN_ID = process.env.CHAIN_ID;
```

Comment:
How does this work?

Reply:
This is basically just ENTITY_MIGRATION renamed to be more clear. If you run without CHAIN_ID then ALL chains will be migrated. If you specify CHAIN_ID then ONLY that chain will be migrated i.e. no need to provide ENTITY_MIGRATION='all'.